core/src/main/java/org/apache/calcite/rex/RexBuilder.java

```diff
@@ -1750,6 +1750,9 @@ public RexNode makeNullLiteral(SqlTypeName typeName) {
    * literal, "SEARCH(arg, SARG([point0..point0], [point1..point1], ...)";
    * otherwise creates a disjunction, "arg = point0 OR arg = point1 OR ...". */
   public RexNode makeIn(RexNode arg, List<? extends RexNode> ranges) {
```

Comment:
This javadoc needs to be fixed to match the behavior of the code.

Reply:
Added javadoc in https://github.com/apache/calcite/pull/3991/commits/32f0485ff2225f48d312566930bb12df384ec7f1.